paasta_tools/cli/schemas/kubernetes_schema.json

```diff
@@ -206,10 +208,14 @@
                                     "exclusiveMinimum": true,
                                     "exclusiveMaximum": false
                                 }
-                            }
+                            },
+                            "required": [
+                                "target_average_value"
```

Comment:
Is this a new requirement? 

What if instead of requiring a value we provided a sane default for users?

Reply:
Sure I can remove it.

Reply:
@solarkennedy after rethinking, I think perhaps we should add it back as required? It is not something people can easily forget since it's the only parameter for cpu/memory metrics. It only needs to be set once, but will make the configuration more readable. People need to check paasta doc everytime if it is hidden.

Reply:
It is certainly a tradeoff between making "hidden" defaults and making users specify all the things.

Please get a second opinion from @EvanKrall